src/content/docs/en/guides/troubleshooting.mdx

```diff
@@ -199,6 +199,29 @@ const answer = sum(2, 4);
 <Debug {answer} />
 ```
 
+## Creating minimal reproductions
+
+When you encounter a bug in Astro, it's most helpful to create a minimal reproduction of the issue. This helps us to understand the context, quickly identify the problem, and fix it.
+
+A minimal reproduction should be a simplified version of the code that reliably demonstrates the issue.
+
+### Create a StackBlitz via [astro.new](https://astro.new)
+
+You can use [astro.new](https://astro.new) to create a new Astro project with a single click. For minimal reproductions, we recommend the minimal (empty) example, so you can isolate the issue.
+
+We ask you to use [StackBlitz](https://stackblitz.com) because it allows us to quickly reproduce your issue without setting up a local environment. With a GitHub repository, we would have to clone it, install dependencies, and run it locally, which can introduce more issues.
+
```

Comment:
You might not be able to reproduce your issue in StackBlitz. For example, your issue might be dependent on a specific environment or package manager, or it may involve HTML Streaming, which isn't supported in StackBlitz. In this case, create a new Astro project using the CLI, reproduce the issue, and upload it to a GitHub repository. Then, link that in the issue.